userguide/docs/faq.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 4
+---
+
+# Frequently Asked Questions
+
+This section aims to answer some common questions as well as covering some pitfalls you might experience when using the NeoForge installer.
+
+## Troubleshooting the NeoForge Installer
+
+For most users the installer should run without error.
+
+If the installer runs but pops up an error dialog box while executing, see the [NeoForge Installer Log File] section below.
+
+### Troubleshooting the NeoForge Installer on Windows
+
+File associations for executing Java applications are sometimes missing or otherwise misconfigured.
+
+For example, running the installer may momentarily flash up a black window, present the `Open with ...` dialog box, or it might open in a file archiving program such as WinRAR or 7-Zip.
+
+To fix this, download and run (double-click) [JarFix], a program designed to fix broken or missing file associations.
+
+Once complete, restart your computer and try running the installer again.
+
+### Troubleshooting the NeoForge Installer on Linux
+
+File associations for executing Java applications are often missing completely on Linux distributions.
+
+In order to run the installer you may need to execute it manually in a terminal.
+
+For instance, with an installer for NeoForge version `21.1.54`, running the following in a terminal should bring up its GUI:
+
+```
+$ java -jar neoforge-21.1.54-installer.jar
+```
+
+For pointers on finding a terminal application on Linux, see the [Locating a Terminal Application] section in the [Checking for and Installing Java] article.
+
+### NeoForge Installer Log File
+
+When experiencing issues with the NeoForge installer, you might be asked to provide its log file, which should be created in the same directory the installer itself resides in.
+
+For example, the installer for version `21.1.54` should create a file with the name of `neoforge-21.1.54-installer.jar.log`.
```

Comment:
This feels oddly out of place. It should be prefaced on where the log file is used. For example, the log file indicates the steps NeoForge took to install, and asking for help anywhere will usually be followed by a question for the log file.